weekly_report.yaml

```diff
@@ -1,17 +1,17 @@
 ---
 teams:
   Practices Team:
-    team_dashboards: 
-      Developer Practices Dashboard: 
-        Filename: DPROD_Report.pdf
-    jira_projects: issues.redhat.com/DPROD
+    team_dashboards:
+      Developer Practices Dashboard:
+        filename: DPROD_Report.pdf
+        url: https://logilica.io/dashboard/023e570d-1398-458b-88e2-2c83ac6ad5c6?variables=W3sidmFyIjoiVGltZSIsInZhbCI6eyJkaW1lbnNpb24iOiJUaW1lIiwiZGF0ZVJhbmdlIjoiZnJvbSA5MCBkYXlzIGFnbyB0byBub3cifX1d&workstream=00000000b5cc5d88
```

Comment:
We need to figure out when and where we want to store configuration file.

Reply:
In that case, do you think I should add onboarded repositories as well?

Reply:
I think that we are going to want, and should, maintain the list of onboarded repositories, and having it in a machine-readable format seems handy, particularly now that we have a "machine" which can read it...so, yes, I think we should keep your version of the configuration file under source control somewhere (and, for convenience and simplicity, we should probably merge it with the one which is here).

Whether we should keep it on GitHub or not is a separate, interesting question...which we should probably discuss in our SUM.